src/Api/OrbitApiVersions.h

```diff
@@ -40,4 +45,29 @@ struct orbit_api_v1 {
   void (*track_double)(const char* name, double value, orbit_api_color color);
 };
 
+// And these are the equivalents built on Linux of the result of building Orbit.h on Windows. They
```

Comment:
nit. This line is a bit confusing to read, consider rephrasing.

Reply:
I rephrased, I hope it's better now.